docs/integrations-in-rancher/rancher-extensions.md

```diff
@@ -67,17 +91,28 @@ You must reload the page after disabling extensions or display issues may occur.
 
 :::
 
-## Rolling Back Extensions
+## Updating and Upgrading Extensions
 
-Under the **Installed** tab, click the **Rollback** button on the extension you wish to roll back.
+1. Click **☰ > Extensions** under **Configuration**.
+1. Select the **Updates** tab. 
+1. Click **Update**.
 
-![Roll back extensions](/img/roll-back-extension.png)
+If there is a new version of the extension, there will also be an **Update** button visible on the associated card for the extension in the **Available** tab.
 
-:::caution
+### Updating and Upgrading Extensions in an Air-gapped Environment
 
-You must reload the page after rolling back extensions or display issues may occur.
+1. Click **☰**, then click on the name of your local cluster.
+1. From the sidebar, select **Workloads > Deployments**.
+1. From the namespaces dropdown menu, select **All Namespaces**.
+1. Find the **cattle-ui-plugin-system** namespace. 
 
-:::
+<!-- TK Waiting for answer -->
```

Comment:
Find the deployment 
ui-plugin-catalog

Click on edit config 
change rancher/ui-plugin-catalog:1.0.1 to rancher/ui-plugin-catalog:1.0.2 (1.0.1 and 1.0.2 are just examples)

click on save.

Reply:
Also, they need to mirror the image rancher/ui-plugin-catalog:1.0.2 to their private registry before dpoing this.